### PR TITLE
Added output of --help to separate readme file to improve searchability

### PR DESCRIPTION
**Pull request by trondhindenes**
This PR adds a new page contianing the full `--help` output in order to improve searchability for possible parameters.

### CODE REVIEW

**Comment by trondhindenes** [comment]
it done

File: cluster-autoscaler/PARAMS.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
+# Full list of parameters
+
+You can grab this list by running:
+`docker run -it gcr.io/google_containers/cluster-autoscaler ./cluster-autoscaler --help`   
+
+```
+Usage of ./cluster-autoscaler:
+      --address string                                                     The address to expose prometheus metrics. (default ":8085")
```
Comment (mwielgus) [inline cluster-autoscaler/PARAMS.md]: Please remove the leading indent and reduce the space between the flag and the description (don't align everything to the longest line)

Comment (mwielgus) [reply cluster-autoscaler/PARAMS.md]: ping

Comment (trondhindenes) [reply cluster-autoscaler/PARAMS.md]: Hi, working on this. I'll have an updated commit later today.

**Comment by mwielgus** [comment]
The update didn't happen later that day (or in the next 2.5 weeks). Closing PR.

**Pull request closed without merging**